tools/crm_mon.c

```diff
@@ -120,7 +120,7 @@ gboolean watch_fencing = FALSE;
 gboolean fence_history = FALSE;
 gboolean fence_full_history = FALSE;
 gboolean fence_connect = FALSE;
-int fence_history_level = 0;
+int fence_history_level = 1;
```

Comment:
I merged a commit from poki making everything static, so you'll want to rebase on current master

Reply:
Btw. this is something that I intentionally left for the future clean-up rounds,
it would be really nice to limit globals (as in symbols) proliferation by compressing
all those bit-wide flags into a single bitfield -- with any non-toy compiler, there should
really be no hit because of some kind of unaligned access, as any modifications shall
boil down to bitmask operations. Feel free to do that right away or I'll do it once this
PR is merged.

Reply:
Agreed that bitmasks are better, but Klaus has some major ongoing projects with crm_mon, so let's leave any refactoring until those are done (which go way beyond just this PR)